client/client_root_validation_test.go

```diff
@@ -22,6 +22,8 @@ type SignedRSARootTemplate struct {
 	RootPem string
 }
 
+var passphraseRetriever = func(string, string, bool, int) (string, bool, error) { return "passphrase", false, nil }
```

Comment:
named arguments please. I have no idea what the `string, string, bool, int` represent. Would probably be good to name the return arguments as well.